documentation/src/main/asciidoc/reference/mapper-orm-bridge-named-predicate.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[mapper-orm-bridge-named-predicate]]
+= Defining named predicates
+
+When implementing a <<mapper-orm-bridge-propertybridge,`PropertyBinder`>>
+or <<mapper-orm-bridge-typebridge,`TypeBinder`>>,
+it is possible to assign "named predicates"
+to index schema elements (either the index root or an <<mapper-orm-bridge-index-field-dsl-object,object field>>).
+
+These named predicates will then be usable as "black boxes" <<search-dsl-predicate-named,through the Search DSL>>,
+referencing them by name and optionally passing parameters.
+
+Below is a simple example using the DSL to declare an object field and assign a named predicate to that field,
+in a property binder.
+
+.Declaring a named predicate
+====
+[source, JAVA, indent=0, subs="+callouts"]
+----
+include::{sourcedir}/org/hibernate/search/documentation/mapper/orm/bridge/namedpredicate/SkuIdentifierBinder.java[tags=binder]
+----
+<1> The binder declares its dependencies on the bridged element, as <<mapper-orm-bridge-propertybridge,usual>>.
+<2> The binder declares the index fields and their type, as <<mapper-orm-bridge-propertybridge,usual>>.
+<3> The binder defines a bridge to populate the fields from the bridged element, as <<mapper-orm-bridge-propertybridge,usual>>.
```

Comment:
If there isn't anything new with that, maybe it is better to remove these points.